src/types.ts

```diff
@@ -54,7 +54,7 @@ export interface PostHogConfig {
     autocapture: boolean | AutocaptureConfig
     rageclick: boolean
     cross_subdomain_cookie: boolean
-    persistence: 'localStorage' | 'cookie' | 'memory' | 'localStorage+cookie'
+    persistence: 'localStorage' | 'cookie' | 'memory' | 'localStorage+cookie' | 'sessionStorage'
```

Comment:
Might be worth aliasing this long and twice repeated `|` list as `type PersistenceType` or such?

Reply:
It's only repeated once as far as I can see, can you give me a pointer?

Reply:
Ah, I suppose the GDRP/OptInOut config? On second thought, it's probably _not_ a good idea to allow that, as those settings SHOULD be persisted. Narrowed that down to just `'cookie' | 'localStorage'| 'localStorage+cookie'`.